dojo/tools/jfrogxray/parser.py

```diff
@@ -95,6 +95,8 @@ def get_item(vulnerability, test):
         title = vulnerability['id'] + " - " + str(cve) + " - " + vulnerability['component']
     else:
         title = str(cve) + " - " + vulnerability['component']
+    component_name = vulnerability['component']
+    component_version = vulnerability['source_comp_id'][len(vulnerability['source_id'])+1:]
```

Comment:
I have not looked nor have tons of experience with this report type, but do we want to rather use (and I know it's not necessarily used in other places) `vulnerability.get('source_comp_id')` to prevent an exception and have a default value instead?